src/js/base/ripe.js

```diff
@@ -30,15 +30,29 @@ ripe.Ripe.prototype.init = function(brand, model, options) {
     this.useDefaults = this.options.useDefaults === undefined ? !this.noDefaults : this.options.useDefaults;
     this.noCombinations = this.options.noCombinations === undefined ? false : this.options.noCombinations;
     this.useCombinations = this.options.useCombinations === undefined ? !this.noCombinations : this.options.useCombinations;
+    this.useSync = this.options.useSync === undefined ? false : this.options.useSync;

```

Comment:
the best option is to instantiate the sync plugin and then add the plugin to the ripe instance, this way there's no relationship between sync and the Ripe instance

Reply:
How should the plugins be exported?
Currently only the Ripe constructor is being exported.

```javascript
var Ripe = ripe.Ripe;
```